planner/cascades/testdata/integration_suite_out.json

```diff
@@ -1098,7 +1098,7 @@
           "Projection_3 10000.00 root  Column#2, Column#3, Column#4, Column#5, Column#6, Column#7, Column#8, Column#9",
           "└─MemTableScan_4 10000.00 root table:PROCESSLIST "
         ],
-        "Result": ["  test Sleep 0 autocommit select USER, HOST, DB, COMMAND, TIME, STATE, INFO, DIGEST from information_schema.processlist 7284ef6cf62f14bccb9fa24aff861b419e9cf59b2c098d3069c7c10bc35d0c14"]
+        "Result": ["  test Sleep 0 autocommit select USER, HOST, DB, COMMAND, TIME, STATE, INFO, DIGEST from information_schema.processlist 786234ab0b57f68ad9452021ba5299ab0c1e879a37c8810ceeaf4a1074fe9daa"]
```

Comment:
Why is it changed?

Reply:
I think it's because we add new key words `DIGEST` and the parser changed